testing/test_server.go

```diff
@@ -15,9 +15,10 @@ import (
 	"github.com/fullstorydev/grpcurl"
 )
 
+// TestServer is the testserver instance
```

Comment:
Not a very helpful comment. If you're going to add a doc comment, please say something a little more useful.